### DISCUSSION TITLE
zk-snarks/5-Pinocchio-Protocol

### DISCUSSION THREAD

**Discussion by giscus** [z2o-k7e/z2o-k7e.github.io#44] [Announcements]
# zk-snarks/5-Pinocchio-Protocol

零知世界, Halo2 Tutorial, world of Zero-knowledge proof, Halo2 教程

https://learn.z2o-k7e.world/zk-snarks/5-Pinocchio-Protocol.html

**Comment by liwenwen896** [comment]
add to 1应该不是加1吧，感觉是多项式的值总和为1？
